site/app/controllers/SelfRejoinController.php

```diff
@@ -129,6 +129,14 @@ private function sendRejoinedStudentEmail($joined_section): void {
             $email = new Email($this->core, $details);
             array_push($emails, $email);
         }
+        $sysadamin_email = $this->core->getConfig()->getSysAdminEmail();
+        if (!empty($sysadamin_email)) {
```

Comment:
`empty()` performs a very loose comparison.  Given that `getSysAdminEmail()` returns a string, you should compare it to the empty string.

```suggestion
        if ($sysadamin_email !== '') {
```

Reply:
I wanted to also check if it's nonnull (which is why I was using null), so I'll make the condition include that too.

Reply:
I see that it's theoretically possible for it to be null, so it can either be left with `empty()`, or a more explicit check for null or the empty string.